src/django_components/components/__init__.py

```diff
@@ -1,3 +1,4 @@
-# flake8: noqa F401
+# NOTE: Components exported here are documented in
+from django_components.components.dynamic import DynamicComponent
```

Comment:
And I will use the exports of this file to generate documentation on pre-defined components.